test/prepare-updates-img

```diff
@@ -14,6 +14,9 @@ for i in ${TEST_RPMS//,/ }; do
     cp $i tmp/rpms
 done
 
+# Copy the payload RPM to the tmp/rpms
+cp "$(ls bots/images/fedora-rawhide-anaconda-payload.rpm)" tmp/rpms/
```

Comment:
This will not read the symlink path: you need: readlink -f bots/images/fedora-rawhide-anaconda-payload

Reply:
@adamkankovsky can you please fix this according to the suggestion? Without readlink we dont get the symlink path

Reply:
Looks like cp -L bots/images/fedora-rawhide-anaconda-payload.rpm tmp/rpms/
works

Reply:
cp -L unfortunately doesn't work as it should